### ISSUE TITLE
Stuck in Read-Only mode?

### ISSUE THREAD

**Issue by batjko** [Unleash/unleash#453]
Hi there,

I have just upgraded to 3.2.11, and now I cannot create any new toggles. The whole thing seems to be read-only all of a sudden, nowhere to click to create anything.

![image](https://user-images.githubusercontent.com/7287076/59138791-f32dd180-8986-11e9-8a14-96817f7ce9ac.png)

Am I blind or is the plus button gone for a reason now? :/

**Comment by ivarconr** [comment]
Do you use adminAuthentication=none?

If yes, this fill fix it:

https://github.com/Unleash/unleash/pull/449

**Comment by batjko** [comment]
> Do you use adminAuthentication=none?

I do, yes. Wasn't an issue before the update, though.

Ok, so for now I've added that function as a preRouterHook, which restored the functionality, similar to this:

```js
const options = {
  preRouterHook: app =>
    app.use('/api/admin/', (req, res, next) => {
      req.user = {
        email: 'foo@bar.baz',
        name: 'dummy',
        permissions: undefined,
        imageUrl: undefined,
      }
      next()
    }),
  adminAuthentication: 'none',
}
async function startServer (options) {
  return unleash.start({ ...options })
}
```

**Comment by ivarconr** [comment]
> I do, yes. Wasn't an issue before the update, though.

I'm sorry about this, dunno why I manage to break this. Thanks for reporting. Will release fix asap.

**Comment by ivarconr** [comment]
released as `3.2.12`.

**Comment by batjko** [comment]
> I'm sorry about this, dunno why I manage to break this. Thanks for reporting. Will release fix asap.

No worries at all. That's some really nice turnaround time for a fix, thank you! 👍
